frontend/src/views/MockExams/MockExamsAttempt.tsx

```diff
@@ -0,0 +1,89 @@
+import {
```

Comment:
Let's clean up the folders a little bit. How about this structure?
![CleanShot 2023-04-29 at 17 34 44](https://user-images.githubusercontent.com/14880213/235310996-3b7c81f6-8954-4898-8b81-320ff7631bb9.png)